src/lm.jl

```diff
@@ -156,7 +156,24 @@ function coeftable(mm::LinearModel)
               ["x$i" for i = 1:size(mm.pp.X, 2)], 4)
 end
 
-predict(mm::LinearModel, newx::Matrix) =  newx * coef(mm)
+# Different methods of prediction for the different factorization types
+getR(x::DensePredChol) = UpperTriangular(x.chol.factors)
+getR(x::DensePredQR) = x.qr[:R]
+
+predict((::LinearModel, newx::Matrix)) = newx * coef(mm)
+
+function predict(mm::LinearModel, newx::Matrix, interval_type::Symbol, level = 0.95)
```

Comment:
From my experience with other statistical software, it's more common to specify alpha than the confidence level, with the implicit understanding that the confidence level is then 1 - alpha.